includes/class-wc-install.php

```diff
@@ -624,6 +624,16 @@ private static function get_schema() {
   PRIMARY KEY (log_id),
   KEY level (level)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}woocommerce_downloadable_product_download_log (
```

Comment:
We have an issue about out long table names causing issues on some hosts. https://github.com/woocommerce/woocommerce/pull/16390 Maybe this should be kept short to save us needing to renaming in the future. `wc_download_log`? cc @claudiosanches